### ISSUE TITLE
linker script file `u-boot/arch/arm/lib/elf_aarch64_efi.lds appears` multiple times

### ISSUE THREAD

**Issue by m01** [m01/rock64-arch-linux-build#17]
I get the following error while trying to build natively (outside Docker) as per current instructions in 4a1587d33fe6ce4e0e1b6cb99f3cbb0160381203. I tried both the Arch Linux toolchain and the Arch Linux ARM one. My old Docker image is still able to build the current sourcecode, so I think this is another build tools/environment issue, rather than an issue with the source code itself.

```
...
/tmp/rock64-arch-linux-build/u-boot/net/nfs.c: In function 'nfs_read_reply':
/tmp/rock64-arch-linux-build/u-boot/net/nfs.c:692:46: warning: taking address of packed member of 'struct rpc_t' may result in an unaligned pointer value [-Waddress-of-packed-member]
  692 |    nfs3_get_attributes_offset(rpc_pkt.u.reply.data);
      |                               ~~~~~~~~~~~~~~~^~~~~
  CC      net/ping.o
  LD      lib/efi_loader/built-in.o
  CC      net/tftp.o
  LD      lib/efi_loader/helloworld.so
aarch64-unknown-linux-gnu-ld.bfd: error: linker script file '/tmp/rock64-arch-linux-build/u-boot/arch/arm/lib/elf_aarch64_efi.lds' appears multiple times
make[4]: *** [scripts/Makefile.lib:382: lib/efi_loader/helloworld.so] Error 1
make[3]: *** [/tmp/rock64-arch-linux-build/u-boot/scripts/Makefile.build:425: lib/efi_loader] Error 2
make[2]: *** [/tmp/rock64-arch-linux-build/u-boot/Makefile:1278: lib] Error 2
make[2]: *** Waiting for unfinished jobs....
  LD      net/built-in.o
make[2]: Leaving directory '/tmp/rock64-arch-linux-build/tmp/u-boot-rock64'
make[1]: *** [Makefile:150: sub-make] Error 2
make[1]: Leaving directory '/tmp/rock64-arch-linux-build/u-boot'
make: *** [Makefile.uboot.mk:10: out/u-boot-rock64/idbloader.img] Error 2
```

**Comment by ydirson** [comment]
You will find this fixed in upstream u-boot by d6507e6fd9e0a6f1a8dd28c18cd320c1f861269e

**Comment by m01** [comment]
Thanks a lot @ydirson, that was a very helpful find.
I've emulated the change you identified and can now build again 🙂
